develop/TodoList/TodoListApp.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TodoListApp.swift
+//  TodoList
+//
+//  Created by Alexander Volokhov
+//
+
+import SwiftUI
+
+@main
+struct TodoListApp: App {
+    
+    @StateObject var listViewModel: ListViewModel = ListViewModel()
```

Comment:
Remove this line
insert and create ViewModel on 18 line.
Don't use environmentObject
On iOS 15 a lot of bugs